websites/E/Emby/presence.ts

```diff
@@ -355,23 +366,43 @@ async function isEmbyWebClient(): Promise<boolean> {
 	return false;
 }
 
+function embyBasenameUrl(): string {
+	return `${`${location.protocol}//${location.host}${location.pathname.replace(
+		location.pathname.split("/").slice(-2).join("/"),
+		""
+	)}`}`;
```

Comment:
Basically `location.origin`?

Reply:
Sort of, `location.origin` could be used there (changed). What the method does is take the basename of the website. Some people install Emby (the service) under a path, such as `https://domain.tld/emby`, the web client is under `/web/index.html` that's why the method removes the last two segments of it.

Some examples:
- Installed under root: `https://domain.tld/web/index.html#!/home.html` retrieves `https://domain.tld/`
- Installed under the path /emby: `https://domain.tld/emby/web/index.html#!/home.html` retrieves `https://domain.tld/emby/`